js/index.js

```diff
@@ -39,4 +39,78 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// Update the img src for the code snippets
+
+let ctaImage = document.getElementById("cta-img");
+ctaImage.setAttribute('src', siteContent["cta"]["img-src"]);
+
+let middleImage = document.getElementById("middle-img");
+middleImage.setAttribute('src', siteContent["main-content"]["middle-img-src"]);
+
+// Update the nav
+
+let navBar = document.querySelector('nav').children;
+navBar[0].textContent = siteContent['nav']['nav-item-1'];
+navBar[1].textContent = siteContent['nav']['nav-item-2'];
+navBar[2].textContent = siteContent['nav']['nav-item-3'];
+navBar[3].textContent = siteContent['nav']['nav-item-4'];
+navBar[4].textContent = siteContent['nav']['nav-item-5'];
+navBar[5].textContent = siteContent['nav']['nav-item-6'];
+
+for (let i = 0; i < 6; i++) {
+  navBar[i].style.color = 'green';
+}
+
+let nav = document.querySelector('nav');
+let newAnchor1 = document.createElement('a');
+let newAnchor2 = document.createElement('a');
+newAnchor1.textContent = 'Blog'
+newAnchor2.textContent = 'Story'
+nav.appendChild(newAnchor1);
+nav.prepend(newAnchor2);
+
+newAnchor1.style.color = 'green';
+newAnchor2.style.color = 'green';
+
+// Update the h1
+
+let title = document.querySelector('h1');
+title.textContent = siteContent['cta']['h1'];
+title.style.width = '230px';
+title.style.textAlign = 'center';
+title.innerHTML = siteContent['cta']['h1'].split('DOM').join('DOM<br>');
```

Comment:
This was probably the best and most secure method to get the desired results in the H1. Great job.